### PR TITLE
removed duplicating parts from license file

### PR DESCRIPTION
**Pull request by azkarmoulana**
All rights reserved had been duplicated in the license file and removed duplicating one necessarily

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PowerShell 6.0
 
-Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (SteveL-MSFT) [inline LICENSE.txt]: If we follow the copyright text in other files, we should keep this one here and remove line 5.

File: LICENSE.txt [hunk 2]
```diff
@@ -27,4 +27,4 @@ SOFTWARE.
 IMPORTANT NOTICE:  THE SOFTWARE ALSO CONTAINS THIRD PARTY AND OTHER
 PROPRIETARY SOFTWARE THAT ARE GOVERNED BY SEPARATE LICENSE TERMS. BY ACCEPTING
 THE LICENSE TERMS ABOVE, YOU ALSO ACCEPT THE LICENSE TERMS GOVERNING THE
-THIRD PARTY AND OTHER SOFTWARE, WHICH ARE SET FORTH IN ThirdPartyNotices.txt
\ No newline at end of file
+THIRD PARTY AND OTHER SOFTWARE, WHICH ARE SET FORTH IN ThirdPartyNotices.txt
```
Comment (SteveL-MSFT) [inline LICENSE.txt]: This line should not be removed

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 2 above)
Comment (azkarmoulana) [reply LICENSE.txt]: @SteveL-MSFT changes done

**Review by SteveL-MSFT — APPROVED** [review]
LGTM.  Thanks for the contribution!

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**